packages/turbo/ios/RNVisitableView.swift

```diff
@@ -8,115 +8,67 @@
 import RNTurboiOS
 import UIKit
 
-class RNVisitableView: UIView, SessionSubscriber {
+class RNVisitableView: UIView, RNSessionSubscriber {
   var id: UUID = UUID()
+  @objc var sessionHandle: NSString? = nil
+  @objc var applicationNameForUserAgent: NSString? = nil
   @objc var url: NSString = "" {
     didSet {
-      if (oldValue != "") {
-        self.controller?.visitableURL = URL(string: String(url))!
-        getSession()?.visit(controller!)
-      }
+      visit()
```

Comment:
When #62 will be ready to merge, we should probably remove this from didSet. I have two suggestions where we can add it:
1. Just move this to visitableWillAppear
2. Delegate making visits to RNSession. We would treat RNSessions as a “another manager for VistiableViews”. RNVisitableView could be treated as a “bridge” between RN and native.